iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    public class CleanupDevicesSample
+    {
+        private const string ImportExportDevicesFileName = "devices.txt";
+
+        private static readonly TimeSpan _waitDuration = TimeSpan.FromSeconds(5);
+        private static readonly IReadOnlyList<JobStatus> s_completedJobs = new[]
+        {
+            JobStatus.Completed,
+            JobStatus.Failed,
+            JobStatus.Cancelled,
+        };
+
+        private readonly RegistryManager _registryManager;
+        private readonly BlobContainerClient _blobContainerClient;
+        private readonly List<string> _deleteDeviceWithPrefix;
+
+        public CleanupDevicesSample(RegistryManager rm, BlobContainerClient sc, List<string> deleteDeviceWithPrefix)
+        {
+            _registryManager = rm ?? throw new ArgumentNullException(nameof(rm));
+            _blobContainerClient = sc ?? throw new ArgumentNullException(nameof(sc));
+
+            _deleteDeviceWithPrefix = deleteDeviceWithPrefix;
+        }
+
+        public async Task RunCleanUpAsync()
+        {
+            // Clean up devices
+            await PrintDeviceCountAsync();
```

Comment:
Does it make sense to print total devices count in hub or the number of devices with delete prefix (or to be deleted)?

Reply:
If you want let's talk offline on this.